scripts/tortoisegit/tortoisegit-uninstall.reg

```diff
@@ -0,0 +1,86 @@
+REGEDIT4
+
+[-HKEY_CLASSES_ROOT\LibraryFolder\background\shellex\ContextMenuHandlers\TortoiseSVN]
```

Comment:
I see I still need to update this file too.

Reply:
Understood. Thanks for catching this.